vpp-manager/config/config.go

```diff
@@ -51,6 +49,24 @@ const (
 	DRIVER_VMXNET3         = "vmxnet3"
 )
 
+var Info = &VppManagerInfo{}
+
+type Link struct {
+	Name string
+	MTU  int
+}
+
+type VppManagerInfo struct {
+	Uplinks          map[int]int //map tapswifindex to linuxmtu
+	Status           int
+	UserSpecifiedMtu int
```

Comment:
As `UserSpecifiedMtu` is the value of `getEnv("CALICOVPP_TAP_MTU")` we should probably pass it in the yaml instead of the json interface status

Reply:
you mean we add an env var `CALICOVPP_TAP_MTU` to agent ?